cinema/models.py

```diff
@@ -30,6 +30,10 @@ class Actor(models.Model):
     def __str__(self):
         return self.first_name + " " + self.last_name
 
+    @property
+    def full_name(self) -> str:
+        return self.first_name + " " + self.last_name
```

Comment:
you can use `return str(self)` here

Reply:
I could, but that would make my result dependent on str. What if someone wanted str to also output id? This is also the reason why we  prefer to use slugrelatedfield instead of stringrelatedfield